src/components/Thanks/ThanksPageShare.vue

```diff
@@ -1,24 +1,6 @@
 <template>
 	<div>
-		<div class="message-content">
-			<div class="row">
-				<div class="thanks__header-h1 tw-text-left large-3 small-1">
-					<kv-material-icon
-						class="tw-w-3 tw-h-3 tw-my-3 tw-align-middle tw-mr-0.5 message-content__success"
-						:icon="mdiCheckAll"
-					/>
-				</div>
-				<div v-if="receipt" class="large-9 small-11 message-content__text">
-					<div v-if="isGuest">
-						<div>Success, we've emailed your receipt to you.</div>
-					</div>
-					<div v-else>
-						<!-- eslint-disable-next-line max-len -->
-						<div>Success, your receipt has been sent to <strong class="fs-mask">{{ lender.email }}</strong></div>
-					</div>
-				</div>
-			</div>
-		</div>
+		<ShareStepper :lender-name="this.lender.firstName" />
```

Comment:
If this is replacing the `Success, your receipt has been sent to` string then the kiva-tests repo will also need to be updated as the presence of this string is used in those automated tests to signify we've reached the thanks page.

Reply:
@christian14b These test updates need to be scoped into this feature work to be merged at the same time.